### PR TITLE
ECPair cleanup

### PR DESCRIPTION
**Pull request by dcousens**

### CODE REVIEW

File: src/ecpair.js [hunk 1]
```diff
@@ -58,20 +58,24 @@ ECPair.fromPublicKeyBuffer = function (buffer, network) {
...
-    var version = buffer[0]
-
     network = network.filter(function (network) {
       return version === network.wif
     }).pop()
 
     if (!network) throw new Error('Unknown network version')
+
+  // network
+  } else if (network) {
```
Comment (fanatid) [inline src/ecpair.js]: `network` always will be truthly here?
why not `} else if (version !== network.wif) throw new Error('Invalid network version')` ?

Comment (dcousens) [inline src/ecpair.js]: Oh true,  it gets defaulted.  Will change

Comment (dcousens) [inline src/ecpair.js]: Fixed in ea4ce4e]

**Comment by fanatid** [comment]
LGTM

**Pull request merged**